roles/node_images_base/vars/packages/suse.yml

```diff
@@ -23,151 +23,118 @@
-  - libbd_mdraid2=2.26-150400.1.5
-  - libcanberra-gtk0=0.30-150400.13.10
-  - libdw1=0.185-150400.5.3.1
-  - libelf-devel=0.185-150400.5.3.1
-  - libelf1=0.185-150400.5.3.1
-  - libfreebl3-hmac=3.79.4-150400.3.29.1
-  - libfreebl3=3.79.4-150400.3.29.1
-  - liblldp_clif1=1.1+44.0f781b4162d3-3.3.1
-  - libopenssl1_1=1.1.1l-150400.7.42.1
-  - libpwquality1=1.4.4-150400.15.4
-  - libstoragemgmt=1.8.5-3.3.1
-  - libsystemd0=249.16-150400.8.25.7
-  - libudev1=249.16-150400.8.25.7
-  - libunwind-devel=1.5.0-4.5.1
+  # rationale: Desired for viewing and dumping small computer system interface (SCSI) devices.
   - lsscsi=0.28-1.24
+  # rationale: Necessary for viewing manuals.
   - man=2.7.6-6.22
-  - nvme-cli-bash-completion=2.0+30.g86f82c58cb97-150400.3.15.1
-  - nvme-cli-zsh-completion=2.0+30.g86f82c58cb97-150400.3.15.1
-  - open-lldp=1.1+44.0f781b4162d3-3.3.1
+  # rationale: Necessary for openscap.
   - openscap-utils=1.3.6-150400.11.3.1
+  # rationale: Necessary for running CVE scans.
   - openscap=1.3.6-150400.11.3.1
-  - openssh-clients=8.4p1-150300.3.18.2
-  - openssh-common=8.4p1-150300.3.18.2
-  - openssh-server=8.4p1-150300.3.18.2
+  # rationale: Necessary for link layer discovery protocol (LLDP) during early startup.
+  - open-lldp=1.1+44.0f781b4162d3-3.3.1
+  # rationale: Necessary for incoming and outgoing secure shells.
   - openssh=8.4p1-150300.3.18.2
-  - openssl-1_1=1.1.1l-150400.7.42.1
-  - perf=5.14.21-150400.44.13.1
-  - perl-File-BaseDir=0.09-bp154.1.25
-  - perl-File-Which=1.22-1.22
-  - perl-MailTools=2.19-1.20
-  - perl-doc=5.26.1-150300.17.11.1
-  - pixz=1.0.6-1.13
-  - postfix=3.5.9-5.9.2
+  # rationale: Necessary for viewing, dumping, and inspecting PCI devices.
+  - pciutils=3.5.6-150300.13.3.1
+  # rationale: Necessary for efficient file transfers.
   - rsync=3.2.3-150400.3.8.1
-  - rsyslog=8.2106.0-150400.5.11.1
-  - screen=4.6.2-5.3.1
+  # rationale: Necessary for inspecting stack traces for applications.
   - strace=5.14-150400.1.7
+  # rationale: Necessary for changing users.
   - sudo=1.9.9-150400.4.26.1
-  - systemd-coredump=249.16-150400.8.25.7
-  - systemd-lang=249.16-150400.8.25.7
-  - systemd-sysvinit=249.16-150400.8.25.7
-  - systemd=249.16-150400.8.25.7
-  - tar=1.34-150000.3.31.1
+  # rationale: Necessary for monitoring and triaging IP connections.
   - tcpdump=4.99.1-150400.1.8
-  - tmux=3.1c-bp152.2.3.1
+  # rationale: Necessary for triaging and debugging packet routes.
   - traceroute=2.0.21-1.29
-  - udev=249.16-150400.8.25.7
-  - unixODBC=2.3.9-150400.14.5
-  - unzip=6.00-150000.4.11.1
+  # rationale: Necessary for viewing, dumping, and inspecting USB devices.
   - usbutils=014-3.3.1
-  - vim-data=9.0.1443-150000.5.43.1
-  - vim=9.0.1443-150000.5.43.1
+  # rationale: Necessary for resolving where an application is actually running from.
   - which=2.21-2.20
-  - yq-bash-completion=4.18.1-bp154.1.17
-  - yq-zsh-completion=4.18.1-bp154.1.17
-  - zip=3.0-2.22
-  # Python
-  - python3-Jinja2=2.10.1-3.10.2
```

Comment:
Future: With the split of the packages role in the next PR, we should have a cloud-init role, and a python role.